db/migrate/20200817151547_add_processed_at_to_digest_runs.rb

```diff
@@ -0,0 +1,5 @@
+class AddProcessedAtToDigestRuns < ActiveRecord::Migration[6.0]
```

Comment:
Minor: the commit message could be more precise:

> we have no evidence that the run was incomplete

Suggest: "not processed", since we do know it's "incomplete" (assuming it was created).

Reply:
Sure, I've updated the message